npm/react18/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@cypress/react18",
+  "version": "0.0.0-development",
+  "description": "Test React components using Cypress",
+  "main": "dist/cypress-react.cjs.js",
+  "scripts": {
+    "build": "rimraf dist && rollup -c rollup.config.js",
+    "postbuild": "node ../../scripts/sync-exported-npm-with-cli.js",
+    "build-prod": "yarn build",
+    "watch": "yarn build --watch --watch.exclude ./dist/**/*"
+  },
+  "devDependencies": {
+    "@cypress/react": "0.0.0-development",
+    "@rollup/plugin-commonjs": "^17.1.0",
+    "@rollup/plugin-node-resolve": "^11.1.1",
+    "cypress": "0.0.0-development",
+    "react": "16.8.6",
```

Comment:
Should we update react deps to 18 for this package? Not sure if these are really even used...

Reply:
I can try but I think this messes up the monorepo dependencies since Lerna hoisting is not perfect. This dependency is not used at all (we grab their React version) but would be nice to have the correct deps if we can.

Reply:
Not really used but going to update it anyway!